src/Std/Classes/Ord.lean

```diff
@@ -216,6 +227,10 @@ theorem TransCmp.lt_of_lt_of_isLE [TransCmp cmp] {a b c : α} (hab : cmp a b = .
   · exact TransCmp.lt_trans hab hbc
   · exact TransCmp.lt_of_lt_of_eq hab hbc
 
+theorem TransCmp.lt_of_lt_of_lt [TransCmp cmp] {a b c : α} (hab : cmp a b = .lt)
```

Comment:
This is just `TransCmp.lt_trans`, and so far neither we nor mathlib have been in the habit of introducing this alias (though I have considered it before).